### ISSUE TITLE
Trees don't decay

### ISSUE THREAD

**Issue by sugary-stardust** [zmatez/wildnature#7]
The mod's trees don't cut down from chopping one block - which is another mod, and it's fine that it doesn't happen - but the mod's tree leaves don't decay... ever. I have Fast Leaf Decay and it doesn't affect the mod's leaves, and the leaves are still there after about two weeks in-game.

**Comment by basicallybea** [comment]
> The mod's trees don't cut down from chopping one block - which is another mod, and it's fine that it doesn't happen - but the mod's tree leaves don't decay... ever. I have Fast Leaf Decay and it doesn't affect the mod's leaves, and the leaves are still there after about two weeks in-game.

Still an issue, and it super sucks. Especially as part of a modded series on YouTube.

**Comment by HentaiHamster** [comment]
yeah same having same issue on 1.14.4. leaves no decay ever and tree dont come down from block. time to burn everything down. lol

**Comment by Gbergz** [comment]
Still an issue!
Any clue if this will be fixed?

**Comment by C8rus** [comment]
1.14.4 tree leaves generate with persistent: true, which prevents them from decaying at all.

**Comment by Gbergz** [comment]
> 1.14.4 tree leaves generate with persistent: true, which prevents them from decaying at all.

Ugh.. I wonder if this is intentional or if it's changing in a future update.

**Comment by Faxmachinen** [comment]
+1 for this issue. It's compounded by above-ground lava lakes starting massive forest fires.
